dv/uvm/core_ibex/scripts/ibex_sim.mk

```diff
@@ -73,10 +73,12 @@ $(comp-results): $(TESTS-DIR)/%/trr.yaml: \
 
 $(METADATA-DIR)/fcov.stamp: $(comp-results) \
   scripts/get_fcov.py
+ifeq ($(COV), 1)
```

Comment:
Nope, this is because `collect_results.py` now uses the coverage results as well as the test results (in order to put them into a report). This is a problem where you do a run that doesn't generate coverage as the step running collect results will never run as it's pre-requisites won't be there.

So I've modified the coverage rules so where we don't enable coverage collection they still run but just output .stamp files that allow the final report generation step to trigger without doing any of the actual coverage merging. I'll add a comment to explain.